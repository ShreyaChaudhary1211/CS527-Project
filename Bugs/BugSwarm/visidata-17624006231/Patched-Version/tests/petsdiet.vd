sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Date		key-col		!	
benchmark	Customer		hide-col		-	
benchmark	Item		hide-col		-	
benchmark	Quantity		type-int		#	
benchmark	SKU		select-col-regex	FOOD	|	
benchmark			dup-selected		"	
benchmark_selectedref	Quantity		aggregate-col	mean	+	
benchmark_selectedref	SKU		freq-col		F	
benchmark_selectedref_SKU_freq	Quantity_mean		type-int		#	
benchmark_selectedref_SKU_freq	Quantity_mean		sort-desc		]	
