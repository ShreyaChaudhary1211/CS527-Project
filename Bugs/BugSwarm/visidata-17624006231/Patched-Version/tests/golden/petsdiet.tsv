SKU	count	Quantity_mean
FOOD219	2	74
FOOD167	1	50
FOOD122	4	6
FOOD213	1	4
FOOD216	1	4
FOOD217	1	4
FOOD215	1	4
FOOD360	1	4
FOOD212	1	4
FOOD211	1	4
FOOD214	1	4
FOOD218	1	4
FOOD170	2	3
FOOD130	3	2
FOOD146	1	2
FOOD121	3	1
FOOD420	1	1
